src/main/java/com/github/terma/jenkins/githubprcoveragestatus/SonarMasterCoverageRepository.java

```diff
@@ -20,27 +22,32 @@
 
     private static final String SONAR_SEARCH_PROJECTS_API_PATH = "/api/projects/index";
     private static final String SONAR_COMPONENT_MEASURE_API_PATH = "/api/measures/component";
-    private static final String SONAR_OVERALL_LINE_COVERAGE_METRIC_NAME = "overall_line_coverage";
+    public static final String SONAR_OVERALL_LINE_COVERAGE_METRIC_NAME = "line_coverage";
```

Comment:
This property is deprecated. 
https://github.com/SonarSource/sonarqube/blob/master/sonar-scanner-engine/src/main/java/org/sonar/scanner/sensor/DefaultSensorStorage.java#L193